clcache.py

```diff
@@ -206,7 +206,7 @@ def computeKey(compilerBinary, commandLine):
         (preprocessedSourceCode, pperr) = preprocessor.communicate()
 
         if preprocessor.returncode != 0:
-            sys.stderr.write(pperr)
+            sys.stderr.write(pperr.decode("utf-8"))
```

Comment:
The commit I deeplinked was altered. Just check https://github.com/frerich/clcache/pull/117